lib/view_models/file_transfer_provider.dart

```diff
@@ -357,10 +359,11 @@ class FileTransferProvider extends BaseModel {
       filename: _filesList[_index].name,
     );
 
-    var _atclient = BackendService.getInstance().atClientInstance;
+    var _atclient = AtClientManager.getInstance().atClient;
     try {
       File file =
-          File(MixedConstants.SENT_FILE_DIRECTORY + _filesList[_index].name);
+          File(MixedConstants.DESKTOP_SENT_DIR + _filesList[_index].name);
```

Comment:
Is this right constant?

Reply:
yes,sent files are stored in the same directory for mobile and desktop.